components/card/styles.ts

```diff
@@ -1,66 +1,106 @@
-import { CardTypes } from '../utils/prop-types'
+import { CardColors, CardVariants } from '../utils/prop-types'
 import { ZeitUIThemesPalette } from '../styles/themes'
 
 export type CardStyles = {
   color: string
   bgColor: string
+  hoverBgColor: string
   borderColor: string
+  hoverBorderColor: string
+}
+
+export type SolidCardStyles = {
+  color: string
+  bgColor: string
+  hoverBgColor: string
+}
+
+export type LineCardStyles = {
+  color: string
+  borderColor: string
+  hoverBorderColor: string
 }
 
 export const getStyles = (
-  type: CardTypes,
+  color: CardColors,
   palette: ZeitUIThemesPalette,
-  isShadow?: boolean,
+  variant: CardVariants,
+  hoverable: boolean,
 ): CardStyles => {
-  const colors: { [key in CardTypes]: Omit<CardStyles, 'borderColor'> } = {
+  const customColor = color
```

Comment:
I think it should be different with solidColors.default.color.

Reply:
The 'color' variable is CardColors Type. And it is the parameter passed in the getStyles method. So I think the user will know the meaning of 'color' variable directly when he passes it to the 'getStyles' method.

Reply:
I worry that other developer will confound the 'color' with solidColors.default.color below. And in card.tsx when using getStyles, 'customColor' is passed. So although props is color, I use customColor instead of color in code.